packages/core/content-type-builder/admin/src/components/Relation/Relation.tsx

```diff
@@ -1,25 +1,30 @@
-/**
- *
- * Relation
- *
- */
-
-import React from 'react';
-
 import { Flex } from '@strapi/design-system';
-import PropTypes from 'prop-types';
-
-import getRelationType from '../../utils/getRelationType';
-
-import RelationField from './RelationField';
-import RelationNaturePicker from './RelationNaturePicker';
 
-const Relation = ({ formErrors, mainBoxHeader, modifiedData, naturePickerType, onChange }) => {
+import { getRelationType } from '../../utils/getRelationType';
+
+import { RelationFormBox } from './RelationField/RelationField';
+import { RelationNaturePicker } from './RelationNaturePicker/RelationNaturePicker';
+
+interface RelationProps {
+  formErrors: Record<string, any>;
+  mainBoxHeader: string;
+  modifiedData: Record<string, any>;
+  onChange: (value: any) => void;
+  naturePickerType: string;
+}
+
+export const Relation = ({
+  formErrors,
+  mainBoxHeader,
+  modifiedData,
+  naturePickerType,
+  onChange,
+}: RelationProps) => {
   const relationType = getRelationType(modifiedData.relation, modifiedData.targetAttribute);
```

Comment:
this should return a string in order to be accepted to RelationFormBox, right now it's either `string` or `undefined`